Sitenotice.php

```diff
@@ -8,21 +8,21 @@
 // Global SiteNotice
 // Increment this version number whenever you change the site notice
 // and don't comment it out
-$wgMajorSiteNoticeID = 56;
+$wgMajorSiteNoticeID = 57;
 
-/* if ( !$wmgSiteNoticeOptOut ) {
+if ( !$wmgSiteNoticeOptOut ) {
 	$wgHooks['SiteNoticeAfter'][] = 'onSiteNoticeAfter'; // show to all users
 
 	function onSiteNoticeAfter( &$siteNotice, $skin ) {
 		global $wmgSiteNoticeOptOut, $snImportant;
 
 		$siteNotice .= <<<EOF
 			<table class="wikitable" style="text-align:center;"><tbody><tr>
-			<td style="font-size:125%">We are aware of issues where some wikis are unable to upload files, or are experiencing issues in thumbnail rendering. We are trying to resolve this as fast as we can, but we don't currently have an estimated time as to when this will be resolved. We apologise for the inconvenience this may cause, and thank you for your patience.</td>
+			<td style="font-size:125%">Miraheze will be performing maintenance on May 20th, 2021 between 20:00 UTC and 20:30 UTC. During this maintenance window, you will not be able to view or upload images or videos.</td>
 			</tr></tbody></table>
 		EOF;
 	}
-} */
+}
 
 // Wrap your sitenotice with <div data-nosnippet>(sitenotice)</div>
```

Comment:
Ahem, excuse me for the "drive away" comment but why wasn't this respected?

Reply:
I'm not sure what you mean? Why wasn't what respected?

Reply:
Because I didn't see that comment until now

Reply:
Done https://github.com/miraheze/mw-config/commit/cc557a4cffca730d7fe14ae080079db737c4305e

Reply:
Thank you.